src/applications/personalization/profile-2/components/Profile2Router.jsx

```diff
@@ -115,7 +129,20 @@ class Profile2Router extends Component {
                 (route.requiresMVI && !this.props.isInMVI)
               ) {
                 return (
-                  <Redirect from={route.path} to="/profile/account-security" />
+                  <Redirect
+                    from={route.path}
+                    to="/profile/account-security"
+                    key={route.path}
+                  />
+                );
+              }
+
+              if (
+                route.path === '/profile/military-information' &&
```

Comment:
I'm going to make that change in this PR since I do agree with those paths coming from a constants file. I believe the `PROFILE_PATHS.MILITARY_INFORMATION` approach should be sufficient